scripts/demo.py

```diff
@@ -1,23 +1,43 @@
 import argparse
 
 import cv2 as cv
+import numpy as np
 import glog as log
 from openvino.inference_engine import IECore
+from dataclasses import dataclass, asdict
 
-from torchdet3d.utils import draw_kp, Regressor, Detector, OBJECTRON_CLASSES
+from torchdet3d.utils import draw_kp, Regressor, Detector, OBJECTRON_CLASSES, IOUTracker
 
+@dataclass
+class SctConfig:
```

Comment:
This config logically belongs to tracker and should be placed nearby it. Also keeping in mind my comment about redundant SCT wrapper this class should be renamed correspondingly.

Reply:
If we do so, the logical meaning of this class will disappear because I wanted to control the parameters of the tracker from outside like from `yml` or python config. But for simplicity, I've created just a config class.

Reply:
Ok, if we had refactored the IOUTracker constructor to take this config, moving this class would have made sense, but for now let's keep it as is.